### PR TITLE
feat(frontend): Add styling for Artifact Node. Partial #6280

### PR DESCRIPTION
**Pull request by zijianjoy**
**Description of your changes:**

Partial #6280

artifactnode

**Checklist:**
- [x] The title for your pull request (PR) should follow our title convention. [Learn more about the pull request title convention used in this repository](https://github.com/kubeflow/pipelines/blob/master/CONTRIBUTING.md#pull-request-title-convention).

### CODE REVIEW

File: frontend/src/components/graph/ArtifactNode.tsx [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+ * limitations under the License.
+ */
+
+import FolderIcon from '@material-ui/icons/Folder';
+import React from 'react';
+import { Handle, Position } from 'react-flow-renderer';
+
+interface ArtifactNodeProps {
+  id: string;
+  data: any;
```
Comment (Bobgy) [inline frontend/src/components/graph/ArtifactNode.tsx:23]: nit: type data?

File: frontend/src/components/graph/ArtifactNode.tsx [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+      <button
+        title={data.label}
+        className='focus:ring flex items-stretch hover:scale-105 transition transform border-0 shadow-lg rounded-lg w-60 h-12'
+      >
+        <div className='flex items-center justify-between w-60 rounded-lg shadow-lg bg-white'>
+          <div className='px-2 flex flex-col justify-center items-center rounded-l-lg'>
+            <FolderIcon className='text-mui-yellow-600' />
+          </div>
+          <div className='flex flex-grow justify-center items-center rounded-r-lg overflow-hidden'>
+            <span className='text-sm truncate ...' id={id}>
```
Comment (Bobgy) [inline frontend/src/components/graph/ArtifactNode.tsx:41]: nit: what is ...?

**Review by Bobgy — COMMENTED** [review]
/lgtm

File: frontend/src/components/graph/ArtifactNode.tsx (hunk 1 above)
Comment (zijianjoy) [reply frontend/src/components/graph/ArtifactNode.tsx:23]: Sounds good, will update this after https://github.com/kubeflow/pipelines/pull/6420 is merged.

File: frontend/src/components/graph/ArtifactNode.tsx (hunk 2 above)
Comment (zijianjoy) [reply frontend/src/components/graph/ArtifactNode.tsx]: I should remove this, good catch!

File: frontend/src/components/graph/ArtifactNode.tsx (hunk 1 above)
Comment (zijianjoy) [reply frontend/src/components/graph/ArtifactNode.tsx]: Updated

**Pull request merged**